packages/components/src/Gesture/withListGesture.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
I don't understand why this file is in a standalone folder, while it have the `List` semantic